examples/misc/tickedstroke_demo.py

```diff
@@ -0,0 +1,101 @@
+"""
+============================
+TickedStroke Patheffect Demo
+============================
+
+Matplotlib's :mod:`.patheffects` can be used to alter the way paths
+are drawn at a low enough level that they can affect almost anything.
+
+The :doc:`patheffects guide</tutorials/advanced/patheffects_guide>`
+details the use of patheffects.
+
+The `~matplotlib.patheffects.TickedStroke` patheffect illustrated here
+draws a path with a ticked style.  The spacing, length, and angle of
+ticks can be controlled.
+
+See also the :doc:`contour demo example
+</gallery/lines_bars_and_markers/lines_with_ticks_demo>`.
+
+See also the :doc:`contours in optimization example
+</gallery/images_contours_and_fields/contours_in_optimization_demo>`.
+"""
+
+import matplotlib.patches as patches
+from matplotlib.path import Path
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.patheffects as patheffects
+
+# Direct to path
+fig, ax1 = plt.subplots(figsize=(6, 6))
+path = Path.unit_circle()
+patch = patches.PathPatch(path, facecolor='none', lw=2, path_effects=[
+    patheffects.withTickedStroke(angle=-90, spacing=10, length=1)])
+
+ax1.add_patch(patch)
+ax1.axis('equal')
+ax1.set_xlim(-2, 2)
+ax1.set_ylim(-2, 2)
+
+plt.show()
+
+###############################################################################
+# Lines and curves with plot and legend
+fig, ax2 = plt.subplots(figsize=(6, 6))
```

Comment:
Same as "Lines with a ticked patheffect". Does it make sense to duplicate this? If so, please copy the updated code from "Lines with a ticked patheffect" to here.

Reply:
In an earlier round, of review, it seemed encouraged to put things in various places where people might find it.  By placing it in lines_with_ticks_demo, we probably reach far more users.  However, keeping it in tickedstroke_demo, it is associated with path effects, so someone looking for what that can do will see another example.

I've tried to make the different variants of this (including the version in test) as similar as possible as of 29fa9c35.